CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
 
 - Add `--npm` flag to `algob init` and `algob unbox`. Note: by default we will use `yarn`.
 - Improved `algob/project-dev-script.sh` which is script setting up a local project.
+- Add `waitRounds` params to `sendAndWait` method. waitRounds have default value is 10.
```

Comment:
```suggestion
- Add `waitRounds` params to `sendAndWait` method. waitRounds default value is 10.
```

Reply:
Please link `sendAndWait` to the API docs

Reply:
Also note that it's an optional argument